server/app/services/cloud/rpc/server_api.rb

```diff
@@ -55,7 +55,7 @@ def put(user_id, path, options)
 
       def require_user(user_id)
         @user = User.find_by(external_id: user_id)
-        raise RpcServer::Error.new(403, 'Forbidden') unless @user
+        raise RpcServer::Error.new(403, 'Cloud user not found. Please try re-login to master with CLI: kontena master login --force') unless @user
```

Comment:
Isn't this a wrong place for these kind of messages? Client that consumes this api should generate proper end-user message based on error code...